package.json

```diff
@@ -84,6 +85,7 @@
 		"npm-run-all": "^4.1.5",
 		"p-all": "^4.0.0",
 		"rimraf": "^3.0.2",
+		"shelljs": "^0.8.5",
```

Comment:
Is that needed? Maybe a leftover?

Reply:
I am eventually planning on using this for getting the tests to run cross platform (see PR description), but yea not needed yet so I will remove 👍